pyproject.toml

```diff
@@ -32,7 +32,7 @@ dependencies = [
   "pyarrow",
   "typing-extensions>=4.6",
   "scipy",
-  "wrapt",
+  "wrapt<1.17",
```

Comment:
This seems to be related to an issue reported [here](https://github.com/Arize-ai/phoenix/discussions/5532), but I haven't been able to reproduce locally.

Reply:
Is it just an issue on Intel?

Reply:
yea, it's just intel mac. so weird

Reply:
probably related to [this](https://github.com/GrahamDumpleton/wrapt/issues/276)